pkgs/development/libraries/libcaca/default.nix

```diff
@@ -11,9 +11,9 @@ stdenv.mkDerivation rec {
     sha256 = "1x3j6yfyxl52adgnabycr0n38j9hx2j74la0hz0n8cnh9ry4d2qj";
   };
 
-  configureFlags = "--disable-x11 --disable-imlib2 --disable-doc";
```

Comment:
That's a bug in `imglib2`. See https://github.com/oxij/nixpkgs/blob/244fba351c341dfc76efe3f1e7a7711368e36edd/pkgs/applications/networking/browsers/w3m/no-x11.patch for how I fixed this in `w3m`.